.eslintrc.js

```diff
@@ -0,0 +1,5 @@
+module.exports = {
+  root: true,
+  extends: ['@react-native', 'plugin:prettier/recommended'],
```

Comment:
question: shouldn't we extend callstack eslint-config?

Reply:
I don't know)

Reply:
Only if it makes sense. It's rather tailored for apps, so not sure whether it would work out nicely. In RN CLI we use `'@react-native-community'`, which is the same as the one used here (just before the rename)